cmd/main.go

```
@@ -58,8 +58,9 @@ func mount(cmd *Cmd, target string) error {
 		err = unpack(f, cmd.MetaDB+".d")
 		if err != nil {
 			log.Error(err)
+		} else {
```

i think if unpack failed u should return the error, not just log and continue!